automol/form/_form.py

```diff
@@ -223,17 +201,18 @@ def from_string(fml_str: str) -> dict[str, int]:
     return fml
 
 
-def sorted_symbols(seq, symbs_first=("C", "H"), symbs_last=()):
+def sorted_symbols(
+    seq: Sequence,
+    symbs_first: Sequence[str] = ("C", "H"),
+    symbs_last: Sequence[str] = (),
+) -> Sequence:
```

Comment:
Correction: `(seq: Sequence[str] | dict[str, int], ....) -> tuple[str, ...]`.

(Note: In general, return types should always use non-generic types like `list[str]` or `tuple[str, ...]`, according to the implementation. Here, it returns a tuple, so we should use `tuple[str, ...]`. The only exception would be if the function is specifically written to be generic and return either a list or tuple, depending on user input, which only happens in a few places.)